builtins: make ST_EstimatedExtent always return NULL

otan opened the pull request:
NULL is always valid for ST_EstimatedExtent, and is the default if
statistics cannot be found for the given column. This allows tools which
use this function to progress.

Refs: #64257
Refs: #52013

Release note (sql change): Stub ST_EstimatedExtent, which always returns
NULL. This allows geoserver to make progress in certain cases, and is a
valid default return value for the function.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/64259)
<!-- Reviewable:end -->

stevendanna commented:
Bazel CI failure should be fixed here: https://github.com/cockroachdb/cockroach/pull/64270

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/64259#-:-MZIXp2d7euh8AhtAOs2:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @otan)

___
*[pkg/sql/sem/builtins/geo_builtins.go, line 5828 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/64259#-MZIXbDr8xKmYM5rEZu4:-MZIXbDr8xKmYM5rEZu5:bxah21e) ([raw file](https://github.com/cockroachdb/cockroach/blob/322b7f1b8cab377d7b3f8ec9a755319665d04317/pkg/sql/sem/builtins/geo_builtins.go#L5828)):*
> ```Go
> 				{"table_name", types.String},
> 				{"geocolumn_name", types.String},
> 				{"parent_only", types.Bool},
> ```

We are always going to ignore `parent_only` yes? If yes, could you update the info comment to say that.


<!-- Sent from Reviewable.io -->

otan committed:
builtins: make ST_EstimatedExtent always return NULL

NULL is always valid for ST_EstimatedExtent, and is the default if
statistics cannot be found for the given column. This allows tools which
use this function to progress.

Release note (sql change): Stub ST_EstimatedExtent, which always returns
NULL. This allows geoserver to make progress in certain cases, and is a
valid default return value for the function.

otan reviewed (commented):
bors r=sumeerbhola

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/64259#-:-MZKeeIaBhFiRMM9hvNX:b-rqwald)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @sumeerbhola)

___
*[pkg/sql/sem/builtins/geo_builtins.go, line 5828 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/64259#-MZIXbDr8xKmYM5rEZu4:-MZKefk5-YkZpJryBgGR:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/322b7f1b8cab377d7b3f8ec9a755319665d04317/pkg/sql/sem/builtins/geo_builtins.go#L5828)):*
<details><summary><i>Previously, sumeerbhola wrote…</i></summary><blockquote>

We are always going to ignore `parent_only` yes? If yes, could you update the info comment to say that.
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->
